### PR TITLE
[CI] Add missed unittests to CI workflow

### PR DESCRIPTION
**Pull request by jiangxin369**
## What is the purpose of the change

Add missed unittests to CI workflow

## Brief change log

*(for example:)*
  - Set timeout of waiting server started to 5 seconds by default
  - Add missed unittests to CI workflow

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

File: ai_flow/endpoint/server/server_config.py [hunk 1]
```diff
@@ -130,4 +131,20 @@ def set_wait_for_server_started_timeout(self, wait_for_server_started_timeout):
...
 
     def get_wait_for_server_started_timeout(self):
-        return self.get('wait_for_server_started_timeout')
+        if 'wait_for_server_started_timeout' in self:
+            return self.get('wait_for_server_started_timeout')
+        else:
+            return 5.0
+
+
+def get_configuration():
```
Comment (Sxnan) [inline ai_flow/endpoint/server/server_config.py:140]: We can use get_configuration method in `settings.py`

File: ai_flow/test/api/ut_workflows/workflows/test_workflow_operation/test_workflow_operation.py [hunk 2]
```diff
@@ -61,6 +63,9 @@ def tearDownClass(cls):
         stop_notification_server(cls.ns_server)
 
     def setUp(self):
+        ai_flow_context.__init_notebook_context_flag__ = False
```
Comment (Sxnan) [inline ai_flow/test/api/ut_workflows/workflows/test_workflow_operation/test_workflow_operation.py:66]: Should set these to false in the unit test who set these to true

File: ai_flow/endpoint/server/server_config.py (hunk 1 above)
Comment (jiangxin369) [reply ai_flow/endpoint/server/server_config.py:140]: Removed

**Review by Sxnan — APPROVED** [review]
LGTM

**Pull request merged**